src/librustdoc/lib.rs

```diff
@@ -13,6 +13,7 @@
 #![feature(ptr_offset_from)]
 #![feature(crate_visibility_modifier)]
 #![feature(never_type)]
+#![feature(str_strip)]
```

Comment:
What is it used for?

Reply:
enables `if let Some(remain) = path_str.strip_prefix("Self::")`

see thead here https://github.com/rust-lang/rust/pull/71289#discussion_r425792312